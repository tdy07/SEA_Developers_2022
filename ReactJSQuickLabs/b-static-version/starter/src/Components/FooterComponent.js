import React from 'react'

function FooterComponent() {
  return (
    <footer className='p-1 text-center'>&copy; QA ltd 2019-2023</footer>
  )
}

export default FooterComponent