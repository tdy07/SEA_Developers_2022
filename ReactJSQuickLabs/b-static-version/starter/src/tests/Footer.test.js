import React from "react"
import { create } from "react-test-renderer"
import FooterComponent from "../Components/FooterComponent"

test('Footer matches snapshot', () => {
    const footer = create(<FooterComponent/>)
    expect(footer.toJSON()).toMatchSnapshot()
})