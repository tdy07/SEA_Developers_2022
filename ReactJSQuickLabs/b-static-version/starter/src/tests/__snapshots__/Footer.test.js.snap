// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer matches snapshot 1`] = `
<footer
  className="p-1 text-center"
>
  Â© QA ltd 2019-2023
</footer>
`;
